front/src/modules/filters-and-sorts/hooks/useActiveFilterCurrentlyEditedInDropdown.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react';
+
+import { useRecoilScopedState } from '@/recoil-scope/hooks/useRecoilScopedState';
+import { TableContext } from '@/ui/tables/states/TableContext';
+
+import { activeTableFiltersScopedState } from '../states/activeTableFiltersScopedState';
+import { tableFilterDefinitionUsedInDropdownScopedState } from '../states/tableFilterDefinitionUsedInDropdownScopedState';
+
+export function useActiveTableFilterCurrentlyEditedInDropdown() {
```

Comment:
I am not convinced by the naming. Right now we have 3 steps in the filter dropdown
- filter select => filter Selected => filter edit => filter Edited => filter apply (which is instantaneous for non searchable fields)

So I would simply call this: selectedDropdownFilter

Reply:
I am actually unclear about this one. Is this a filter which is already applied to the table x edited in Dropdown? If yes, not sure why we need it!

Reply:
Yes exactly it is for having behaviors like changing only the operand without having to enter the rest again